expected/12.16/types/macaddr6.out

```diff
@@ -0,0 +1,864 @@
+--Testcase 125:
+ALTER FOREIGN TABLE "type_MACADDR" ALTER COLUMN "m" OPTIONS (SET column_type 'int');
+--Testcase 126:
+INSERT INTO "type_MACADDR" ("i", "m") VALUES (62, '03:00:00:00:00:00');
+--Testcase 127:
+ALTER FOREIGN TABLE "type_MACADDR" ALTER COLUMN "m" OPTIONS (SET column_type 'TEXT');
+--Testcase 128:
+INSERT INTO "type_MACADDR" ("i", "m") VALUES (63, '00:00:00:00:00:01');
+--Testcase 129:
+ALTER FOREIGN TABLE "type_MACADDR" ALTER COLUMN "m" OPTIONS (SET column_type 'BLOB');
+--Testcase 130:
+INSERT INTO "type_MACADDR" ("i", "m") VALUES (64, '00:00:00:00:00:02');
+--Testcase 131:
+ALTER FOREIGN TABLE "type_MACADDR" ALTER COLUMN "m" OPTIONS (SET column_type 'int');
+--Testcase 132:
+INSERT INTO "type_MACADDR" ("i", "m") VALUES (65, '00:00:00:00:00:03');
+--Testcase 133:
+ALTER FOREIGN TABLE "type_MACADDR" ALTER COLUMN "m" OPTIONS (SET column_type 'TEXT');
+--Testcase 134:
+INSERT INTO "type_MACADDR" ("i", "m") VALUES (66, '00:00:01:00:00:00');
+--Testcase 135:
+ALTER FOREIGN TABLE "type_MACADDR" ALTER COLUMN "m" OPTIONS (SET column_type 'BLOB');
+--Testcase 136:
+INSERT INTO "type_MACADDR" ("i", "m") VALUES (67, '00:00:02:00:00:00');
+--Testcase 137:
+ALTER FOREIGN TABLE "type_MACADDR" ALTER COLUMN "m" OPTIONS (SET column_type 'int');
+--Testcase 138:
+INSERT INTO "type_MACADDR" ("i", "m") VALUES (68, '00:00:03:00:00:00');
+--Testcase 139:
+SELECT * FROM "type_MACADDR" ORDER BY "m" ASC;
+ i  |         m         
+----+-------------------
+ 63 | 00:00:00:00:00:01
+ 64 | 00:00:00:00:00:02
+ 65 | 00:00:00:00:00:03
+ 66 | 00:00:01:00:00:00
+ 67 | 00:00:02:00:00:00
+ 68 | 00:00:03:00:00:00
+ 60 | 01:00:00:00:00:00
+ 61 | 02:00:00:00:00:00
+ 62 | 03:00:00:00:00:00
+(9 rows)
+
+--Testcase 140:
+SELECT * FROM "type_MACADDR" ORDER BY "m" DESC;
+ i  |         m         
+----+-------------------
+ 62 | 03:00:00:00:00:00
+ 61 | 02:00:00:00:00:00
+ 60 | 01:00:00:00:00:00
+ 68 | 00:00:03:00:00:00
+ 67 | 00:00:02:00:00:00
+ 66 | 00:00:01:00:00:00
+ 65 | 00:00:00:00:00:03
+ 64 | 00:00:00:00:00:02
+ 63 | 00:00:00:00:00:01
+(9 rows)
+
+--Testcase 150:
+CREATE FOREIGN TABLE "type_MACADDRpk" (col macaddr OPTIONS (key 'true')) SERVER sqlite_svr;
+--Testcase 151:
+ALTER FOREIGN TABLE "type_MACADDRpk" ALTER COLUMN col OPTIONS (ADD column_type 'BLOB');
+--Testcase 152:
+INSERT INTO "type_MACADDRpk" VALUES ('01:02:03:04:05:06');
+--Testcase 153:
+ALTER FOREIGN TABLE "type_MACADDRpk" ALTER COLUMN col OPTIONS (SET column_type 'int');
+--Testcase 154: NO ERR, but the same semantics!
+INSERT INTO "type_MACADDRpk" VALUES ('01:02:03:04:05:06');
+--Testcase 155:
+ALTER FOREIGN TABLE "type_MACADDRpk" ALTER COLUMN col OPTIONS (SET column_type 'text');
+--Testcase 156: NO ERR, but the same semantics!
+INSERT INTO "type_MACADDRpk" VALUES ('01:02:03:04:05:06');
+--Testcase 157:
+ALTER FOREIGN TABLE "type_MACADDRpk" ALTER COLUMN col OPTIONS (ADD column_type 'BLOB');
+ERROR:  option "column_type" provided more than once
```

Comment:
I think this is an incorrect error. `OPTIONS` should be `SET column_type `BLOB`.

Reply:
Fixed across all versions. Please ensure, @son-phamngoc !

Reply:
Confirmed. Thanks. Please help me to resolve this comment.